bio/samtools/fastx/wrapper.py

```
@@ -8,12 +8,17 @@
 
 samtools_opts = get_samtools_opts(
-    snakemake, parse_write_index=False, parse_output_format=False
+    snakemake, parse_write_index=False, parse_output=False, parse_output_format=False
 )
 extra = snakemake.params.get("extra", "")
 log = snakemake.log_fmt_shell(stdout=True, stderr=True)
 
 
+# Parse outputs
+fq_out = ""
+for label, out_file in snakemake.output.items():
```

Ruff (0.15.21)

[error] Undefined name `snakemake`

(F821)